tide/manage.go

```
@@ -442,6 +442,9 @@ func (m *manager) Delete() error {
 		}
 		return true, nil
 	})
+	if err != nil {
+		return err
+	}
```

@ntfrnzn was this meant to be best effort only?

---

no, let's return the error (until we decide otherwise)